homeworks/sasha.demidenko_SashaDemi/5-logic/homework.js

```diff
@@ -14,6 +14,8 @@ function calculate(firstValue, secondValue, operation) {
         return ('Enter a number');
     } if (!Number.isFinite(secondNumber)) {
         return ('Enter a number');
+    } if (firstNumber || secondNumber === 0) {
```

Comment:
Nah)
Your error is shown for a pretty legal case
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/14363dd4-1f42-4b80-bf16-7377ec3c7bcc)

Reply:
You should return error 'Enter a number' for the case, when `firstValue` or `secondValue` is an empty string

Reply:
Also, if you return same value for these two conditions, you can combine them in one `OR` condition:
```js
if (!Number.isFinite(firstNumber)) {
        return ('Enter a number');
    } if (!Number.isFinite(secondNumber)) {
        return ('Enter a number');
}
```